tests/pr_test.go

```diff
@@ -61,21 +63,29 @@ func TestRunUpgradeExample(t *testing.T) {
 	}
 }
 
-func TestRunFSCloudExample(t *testing.T) {
+func TestRunFSCloudDAExample(t *testing.T) {
 	t.Parallel()
 
-	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
-		Testing:       t,
-		TerraformDir:  "examples/fscloud",
-		Prefix:        "fscloud",
-		Region:        "us-south", // For FSCloud locking into us-south since that is where the dedicated host is provisioned
-		ResourceGroup: resourceGroup,
-		TerraformVars: map[string]interface{}{
-			"access_tags": permanentResources["accessTags"],
-			// crn of the dedicated host
-			"environment_crn": permanentResources["dedicatedHostCrn"],
-		},
+	// Generate a 6 char long random string for the instance_name
+	randomBytes := make([]byte, 6)
+	_, err := rand.Read(randomBytes)
+	randomInstanceName := "fscloud" + base64.URLEncoding.EncodeToString(randomBytes)[:6]
```

Comment:
Do we need this? Can we not pass `Prefix: "fscloud",` into the setup options, and then reference it like options.Prefix ? I think it should then return a randomly generated prefix value?